config/phpstan.services.neon

```diff
@@ -23,3 +23,13 @@ services:
         class: \FiveLab\Component\CiRules\PhpStan\ForbiddenPassArgumentAsReferenceRule
         arguments: [ PhpParser\Node\Stmt\ClassMethod ]
         tags: [ phpstan.rules.rule ]
+
+    -
+        class: \FiveLab\Component\CiRules\PhpStan\MethodCallConsistencyRule
+        arguments:
+            - @reflectionProvider
```

Comment:
Use only one parameter, so that the code is the same  please use one line for arguments.